tests/store/crud/test_update.py

```diff
@@ -366,3 +368,111 @@ def test_update_analysis_from_slurm_run_status(
 
     # THEN it should update the analysis jobs
     assert updated_analysis.jobs
+
+
+def test_update_tower_jobs(analysis_store: MockStore, tower_jobs: List[dict], case_id: str):
+    """Assess that jobs are successfully updated when using NF Tower."""
+
+    # GIVEN an analysis without failed jobs
+    analysis: Optional[Analysis] = analysis_store.get_latest_analysis_for_case(case_id=case_id)
+    assert not analysis.jobs
+
+    # WHEN analysis jobs are deleted
+    analysis_store.delete_analysis_jobs(analysis=analysis)
+
+    # THEN the analysis object should have no failed jobs
+    assert not analysis.jobs
```

Comment:
Is this part of the test necessary? It is already stated that the store has no jobs.

Reply:
Fixed